api-tests/core/admin/ee/review-workflows.test.api.js

```diff
@@ -174,7 +174,7 @@ describeOnCondition(edition === 'EE')('Review workflows', () => {
   });
 
   describe('Create workflow', () => {
-    test('You can not create a workflow without stages', async () => {
+    test('It should create a workflow without stages', async () => {
```

Comment:
Should this be 'It should not...' ?

Reply:
true :) Now that I see it I am not following that pattern in this test file, I will update all of them